### ISSUE TITLE
Error on first cell

### ISSUE THREAD

**Issue by jlnhlfan** [tg-bomze/BabyGAN#20]
I really wanted to use this after not using it for a while, but the very first cell already has an error; Tensorflow1 is unsupported on Colab. This renders the Colab unusable.

Since I’m not a coder, what solutions are available to me, if any?

**Comment by vdruts** [comment]
Yep, came here to post this. ValueError: Tensorflow 1 is unsupported in Colab.

**Comment by vdruts** [comment]
Update. Other cells have issues. None of it works.

**Comment by jlnhlfan** [comment]
This isn't even usable anymore! To tg-bomze... please update this so it's actually functional.
